setup.cfg

```diff
@@ -0,0 +1,10 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
If we're using setup.cfg, let's put everything in it. Like so https://github.com/DataDog/datadogpy/pull/475

Reply:
done